sigmastate/src/test/scala/sigmastate/helpers/SigmaTestingCommons.scala

```diff
@@ -199,11 +198,11 @@ trait SigmaTestingCommons extends PropSpec
     r.positionLimit shouldBe positionLimitBefore
 
     // using ergo's(scorex) reader/writer
-    val w = new VLQByteStringWriter()
-    serializer.serializeWithGenericWriter(v, w)
-    val byteStr = w.result()
-    byteStr.nonEmpty shouldBe true
-    serializer.parseWithGenericReader(new VLQByteStringReader(byteStr)) shouldEqual v
+//    val w = new VLQByteStringWriter()
```

Comment:
commented out code is not good. Whether remove it or replace with another one.

Reply:
removed.